ui/src/containers/Personas/PersonasImports/stages/ColumnHeaderEditor/index.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { compose } from 'recompose';
+import withStyles from 'isomorphic-style-loader/lib/withStyles';
+import styles from './styles.css';
+import InputFields from './InputFields';
+
+export const ColumnHeaderEditor = ({
+  model, // personasImports model
+  disabled
+}) => (
+  <table className={styles.table}>
+    <thead>
+      <tr className={styles.tr}>
+        <th className={styles.th}>ID Names</th>
```

Comment:
I think this might be clearer, what do you think?
```suggestion
        <th className={styles.th}>CSV Column Name</th>
```